dataland-keycloak/realms/datalandsecurity-realm.json

```diff
@@ -639,20 +640,36 @@
     "notBefore" : 0,
     "groups" : [ ]
   }, {
-      "id" : "d5fcac69-9e77-4b91-9295-609f41676cc0",
-      "createdTimestamp" : 1702466529529,
-      "username" : "service-account-dataland-batch-manager",
-      "enabled" : true,
-      "totp" : false,
-      "emailVerified" : false,
-      "serviceAccountClientId" : "dataland-batch-manager",
-      "credentials" : [ ],
-      "disableableCredentialTypes" : [ ],
-      "requiredActions" : [ "terms_and_conditions" ],
-      "realmRoles" : [ "ROLE_UPLOADER", "default-roles-datalandsecurity" ],
-      "notBefore" : 0,
-      "groups" : [ ]
-    }, {
+    "id" : "d5fcac69-9e77-4b91-9295-609f41676cc0",
+    "createdTimestamp" : 1702466529529,
+    "username" : "service-account-dataland-batch-manager",
+    "enabled" : true,
+    "totp" : false,
+    "emailVerified" : false,
+    "serviceAccountClientId" : "dataland-batch-manager",
+    "credentials" : [ ],
+    "disableableCredentialTypes" : [ ],
+    "requiredActions" : [ "terms_and_conditions" ],
+    "realmRoles" : [ "ROLE_UPLOADER", "default-roles-datalandsecurity" ],
+    "notBefore" : 0,
+    "groups" : [ ]
+  }, {
+    "id": "59188f43-43c1-44a7-98cb-33065f1b0a43",
+    "createdTimestamp": 1707208162593,
+    "username": "service-account-dataland-community-manager",
+    "enabled": true,
+    "totp": false,
+    "emailVerified": false,
+    "serviceAccountClientId": "dataland-community-manager",
+    "credentials" : [ ],
+    "disableableCredentialTypes": [],
+    "requiredActions": [
+      "terms_and_conditions"
+    ],
+    "realmRoles" : ["ROLE_REVIEWER", "ROLE_UPLOADER" ,"default-roles-datalandsecurity"],
```

Comment:
Compared to the batch manager, why does the community manager need to have the reviewer role?

Reply:
I have no idea how that got here and I highly doubt that it is necessary. ROLE_USER should be sufficient.

Reply:
reverted to default-roles-datalandsercurity only. It should include ROLE_USER

Reply:
Nope. This is not auto generated AFAIK. But I see why the reviewer role is needed now. The message that triggers our new method to check if a request is fulfilled by a fresh dataset races against the setting of the qaStatus to accepted/rejected in the backend. Thus our method might try to get metadata for a dataset that is still pending which is forbidden for standard users but allowed for reviewers.